.github/workflows/release.yml

```diff
@@ -270,9 +270,89 @@ jobs:
           npm run prepare
           npm publish
 
+  publish_admin:
+    name: ${{ matrix.name }}
+    needs: [get_version_code, publish_nuget]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["macOS", "windows-os"]
+        include:
+          - name: macOS
+            os: macOS-13
+            version: "14.2"
+            osFolder: "maccatalyst"
+          - name: windows-os
+            os: windows-2022
+            version: "latest"
+            osFolder: "windows"
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Change Directory
+        run: cd ${{ github.workspace }}/src/electionguard-ui/ElectionGuard.UI
+
+      - name: Wait For Nuget Package
+        run: |
+          for i in {1..60}
```

Comment:
needed ot handle the case where there are changes in both the core encryption lib and the admin app, so we wait for the nuget package to get pushed, and wait till its indexed by nuget which can take a few minutes. this way we are guaranteeing that the ui build version matches the sdk version. This link could be broken if we need to or if this is too brittle.

Reply:
Agreed, since the UI is taking it from nuget directly, that could take quite a few minutes to get resolved.